infisical_sdk/infisical_requests.py

```
@@ -49,6 +67,42 @@ def from_dict(cls, data: Dict) -> 'APIResponse[T]':
+    if network_errors is None:
+        network_errors = NETWORK_ERRORS
+        
+    def decorator(func: Callable) -> Callable:
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            retry_count = 0
+            
+            while True:
+                try:
+                    return func(*args, **kwargs)
+                except tuple(network_errors) as error:
```

Ruff (0.8.2)

Local variable `error` is assigned to but never used

Remove assignment to unused variable `error`

(F841)